src/Notify.vue

```diff
@@ -1,9 +1,16 @@
 <template>
-  <div class="notify" :class="'notify-' + options.position">
+  <div class="notify" :class="'notify-' + options.position" :style="{ width: width}">
     <transition-group name="notify" tag="div" @enter="slideDown" @leave="slideUp">
       <div v-for="(item, key) in items" :key="key" class="notify-item">
         <div :class="item.options.itemClass">
-          <span :class="item.options.iconClass" v-if="item.options.iconClass"></span> {{ item.text }}</div>
+          <button v-if="item.options.closeButton === 'bulma'" class="delete" @click="removeItem(key)"></button>
```

Comment:
I do not like the strict tie-in to bulma (or bootstrap for that matter), isn't there another solution that will give the same results?

Reply:
Maybe something like that:
```
<button v-if="item.options.closeButtonClass" :class="item.options.closeButtonClass" @click="removeItem(key)" type="button"></button>
            <span aria-hidden="true">&times;</span>
          </button>
```
If closeButtonClass option === false (default false) - there is no close button
Would this be good?

Reply:
Jup, that would be a lot better :-)

Reply:
@websmurf done